RELEASE_NOTES.md

```diff
@@ -30,6 +30,9 @@ v1.0.0-alpha.xx
   module. These have been replaced by domain-specific traits and
   specifications defined in their own repositories, such as
   [OpenAssetIO-MediaCreation](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation).
+  Renamed remaining constants to have an `InfoKey_` prefix and migrated
```

Comment:
[fix] Can we update documentation and code references to the new names?

Reply:
Whoops! Loads of places missed. Fixed in 165dd87f